src/Handlers/CardsPaymentMethodHandler.php

```diff
@@ -26,8 +26,11 @@
 namespace Adyen\Shopware\Handlers;
 
 use Shopware\Core\Checkout\Payment\Cart\PaymentHandler\AsynchronousPaymentHandlerInterface;
+use Shopware\Core\Checkout\Payment\Cart\PaymentHandler\PreparedPaymentHandlerInterface;
 
-class CardsPaymentMethodHandler extends AbstractPaymentMethodHandler implements AsynchronousPaymentHandlerInterface
+class CardsPaymentMethodHandler extends AbstractPaymentMethodHandler implements
+    AsynchronousPaymentHandlerInterface,
+    PreparedPaymentHandlerInterface
```

Comment:
Can we declare this implements only once at `AbstractPaymentMethodHandler` where we actually implement the interface instead of repeating for every handler?

Reply:
good idea, I meant to do this in a previous PR but somehow this time I didn't even think about it 😄

Reply:
Done in 12ac9b3